src/containers/Flow/FlowList/FlowList.tsx

```diff
@@ -137,18 +139,22 @@ export const FlowList = () => {
       icon: configureIcon,
       parameter: 'uuid',
       link: '/flow/configure',
+      isMoreOption: false,
```

Comment:
have a default value for isMoreOption. Provide value only where its different from the default

Reply:
Done